services/spar/src/Spar/Error.hs

```diff
@@ -296,6 +296,8 @@ mapScimSubsystemErrors =
         Scim.notFound "group members" badIds
       ScimSubsystemScimGroupWithNonScimMembers badIds ->
         Scim.badRequest Scim.InvalidValue (Just $ "These users are not \"managed_by\" = \"scim\": " <> renderIds badIds)
+      ScimSubsystemInternalError _ ->
```

Comment:
i think there is also a scim error for that ("serverError" or something?), so you can use ScimSubsystemError here as well.

Reply:
I'm not sure to understand, it's defined as:

```haskell
serverError ::
  -- | Error details
  Text ->
  ScimError
serverError details =
  ScimError
    { schemas = [Error20],
      status = Status 500,
      scimType = Nothing,
      detail = pure details
    }
```